### DISCUSSION TITLE
不会弄头像啊!!

### DISCUSSION THREAD

**Discussion by FIREFLY26710-WEB** [share121/inter-knot#918] [其他]
![mmexport1722052485112](https://github.com/user-attachments/assets/26610db4-afa8-46c4-ba3a-91a9001e5a96)
就这个，多好看，但我添加了没有反应
球球各位绳匠指点一下（哭）

**Comment by Elysia-lover** [comment]
这不已经换上了吗，换好等一会就行了

**Comment by Dave-12138** [comment]
![Screenshot_20240810_185627_com.microsoft.emmx.canary.png](https://github.com/user-attachments/assets/792a22dd-5a00-44cd-9a95-76ee065e784b)

对啊这不都换上了吗

**Comment by FIREFLY26710-WEB** [comment]
学会了各位
